samples/dashboards/edit-mode/toolbars-options/demo.js

```diff
@@ -0,0 +1,47 @@
+Dashboards.board('container', {
+    editMode: {
+        enabled: true,
+        contextMenu: {
+            enabled: true
+        }
+    },
+    gui: {
+        layouts: [{
+            id: 'layout-1',
+            rows: [{
+                editMode: {
+                    hiddenToolbarItems: ['destroy']
+                },
+                cells: [{
+                    id: 'dashboard-col-0',
+                    editMode: {
+                        hiddenToolbarItems: ['settings', 'destroy']
```

Comment:
Why is it `hiddenToolars` instead of toolbars property that covers which elements should be visible (by default all of them).

It is the same solution like we have in context menu items.

Reply:
You mean sth like this:
```ts
toolbarItems: {
    settings: {
        enabled: false
    },
    destroy: {
        enabled: false
    }
},
```
I like it more than array @ddragula what do you think?

Reply:
The option with an array of disabled elements was the fastest method for workaround needs, here we can do something more user-friendly indeed. I like the proposition.

Reply:
I like the suggested idea 👍